app/Http/Controllers/ApplicationController.php

```diff
@@ -16,20 +16,31 @@ class ApplicationController extends \Controller
     'activities'      => 'required',
     'essay1'          => 'required',
     'essay2'          => 'required',
+    'extra_question_1'=> 'sometimes|required',
```

Comment:
Laravel has an answer for everything! 👏

Reply:
My exact thought when I got the google result for this!